### PR TITLE
Add ice status uavcan bridge and mavlink stream

### PR DESCRIPTION
**Pull request by PonomarevDA**
**Describe problem solved by this pull request**
The problem is described here #17861. It is related at least to possibility of monitoring internal combustion engine state from QGC, but also could be a first step of ICE control implementation.

**Describe your solution and possible alternatives**
Here I added uorb msg, simple uavcan bridge and mavlink stream ice_status -> EFI_STATUS.
Uorb message is based on UAVCAN msg [ice_status](https://legacy.uavcan.org/Specification/7._List_of_standard_data_types/#status-4). Here I suggested that CylinderStatus size = 16 is to much for uorb msg, so I keep fields only for a single CylinderStatus.
Also I made an assumption that 10 Hz will be enough. Higher frequency of this UAVCAN msg could significantly load the bus.

**Test data / coverage**
I made only a simple test by sending fake UAVCAN message to cuav V5.

Here what I send to UAVCAN:
image

Here what I have on uorb:
image

Here what I have on QGC: 
image

So, it is exactly what I expected.

### CODE REVIEW

File: src/modules/mavlink/mavlink_main.cpp [hunk 1]
```diff
@@ -1550,6 +1550,7 @@ Mavlink::configure_streams_to_default(const char *configure_single_stream)
 		configure_stream_local("VFR_HUD", 4.0f);
 		configure_stream_local("VIBRATION", 0.1f);
 		configure_stream_local("WIND_COV", 0.5f);
+		configure_stream_local("EFI_STATUS", 2.0f);
```
Comment (dagar) [inline src/modules/mavlink/mavlink_main.cpp:1553]: Can you keep the list sorted alphabetically?

File: msg/CMakeLists.txt [hunk 2]
```diff
@@ -83,6 +83,7 @@ set(msg_files
 	heater_status.msg
 	home_position.msg
 	hover_thrust_estimate.msg
+	ice_status.msg
```
Comment (dagar) [inline msg/CMakeLists.txt:86]: Nitpicky, but is "ice" a well known abbreviation? I think I'd prefer just spelling it out so at a glance the the log is obvious.

**Comment by dagar** [comment]
I'd also suggest adding it to logger by default at a minimal rate.

https://github.com/PX4/PX4-Autopilot/blob/76b34132a5c1df4f6ef82304724b9e723f0dadce/src/modules/logger/logged_topics.cpp#L46

File: src/modules/mavlink/mavlink_main.cpp (hunk 1 above)
Comment (PonomarevDA) [reply src/modules/mavlink/mavlink_main.cpp]: fixed

File: msg/CMakeLists.txt (hunk 2 above)
Comment (PonomarevDA) [reply msg/CMakeLists.txt]: fixed

**Comment by PonomarevDA** [comment]
I also added internal_combustion_engine_status to uorb_rtps_message_ids.yaml to fix rtps problem.
According to remaining failed checks, it's not enough flash memory for fmu_v2 and smth wrong with EKF.
Do you have any ideas how to fix them?

**Comment by PonomarevDA** [comment]
Problem with EKF was fixed by merging master branch. As I understand it was happened because my version had ekf as submodule, but master branch now has just source files. Not enough flash memory for fmu_v2 was solved after merging as well.
I also fixed merge conflict in `uorb_rtps_message_ids.yaml`.

**Review by dagar — APPROVED** [review]

**Pull request merged**